scripts/python/PyAutoscoper/connect.py

```diff
@@ -413,6 +422,7 @@ def optimizeFrame(
         self,
         volume,
         frame,
+        guess,
```

Comment:
Should this parameter be introduced last and specified as `guess=InitialGuess.PREVIOUS_FRAME` ? 

It seems this would be consistent with the behavior that was implemented with the following:

```
            if frame != 0:
                pose = self.getPose(volume=volume, frame=(frame - 1))
                self.setPose(volume=volume, frame=frame, pose=pose)
```

Reply:
Before I was leaving the function `optimizeFrame` with no default values and instead populating the function `trackingDialog` with all of the default values from the tracking dialog in the UI. This would be `InitialGuess.CURRENT_FRAME` to mirror the UI.

https://github.com/BrownBiomechanics/Autoscoper/blob/69ecd3c331ef3aaed516bb4d8b26884c77df3784/scripts/python/PyAutoscoper/connect.py#L515-L528